test/lib/heroku-api-test.js

```diff
@@ -57,9 +57,10 @@ describe('heroku-api', function () {
       const pipeline = '123-abc'
       const number = 1
       const testRun = { number }
-      const api = nock(`https://api.heroku.com`)
+      const api = nock(`https://api.heroku.com`, {reqheaders: {Accept: 'application/vnd.heroku+json; version=3.ci'}})
         .get(`/pipelines/${pipeline}/test-runs/${number}`)
-        .matchHeader('Accept', 'application/vnd.heroku+json; version=3.ci')
+        // nock node 8 bug https://github.com/node-nock/nock/issues/925
+        // .matchHeader('Accept', 'application/vnd.heroku+json; version=3.ci')
```

Comment:
I'd almost rather not have these comments at all. The new implementation seems fine, so I doubt we'd ever come back to this syntax.